consensus/types/src/data_column_sidecar.rs

```diff
@@ -128,8 +128,11 @@ impl<E: EthSpec> DataColumnSidecar<E> {
         let blob_cells_and_proofs_vec = blobs
             .into_par_iter()
             .map(|blob| {
-                let blob = KzgBlob::from_bytes(blob).map_err(KzgError::from)?;
-                kzg.compute_cells_and_proofs(&blob)
+                let blob = blob
+                    .as_ref()
+                    .try_into()
+                    .expect("blob should have a guaranteed size due to FixedVector");
+                kzg.compute_cells_and_proofs(blob)
```

Comment:
I think this is the main change that fixes the stack overflow on `das` testing right?

The other changes change mainnet code path - it's probably not something we can merge to the `unstable` branch now, although I think we do want them eventually when `c-kzg` adopts it or when `rust-eth-kzg` is production-ready for mainnet.

I think we can merge this branch to `das` and I'll just cherry-pick the changes relevant to `compute_cells_and_proofs` onto `unstable`.

Reply:
Yep this is correct